### PR TITLE
Bugfix/popup blocker prevents generated codes

### LINKED ISSUE #51: Popup blocker prevents generated codes window to open

### PR DESCRIPTION
**Pull request by lindskogen**
Closes #51

### CODE REVIEW

File: src/client/Admin.js [hunk 1]
```diff
@@ -66,14 +66,9 @@ var Admin = React.createClass({
         });
     },
     confirmOpenPrintPage() {
+        this.getServerStatus();
```
Comment (Dr-Horv) [inline src/client/Admin.js]: Why do we ned to get server status here?

File: src/client/backend.js [hunk 2]
```diff
@@ -18,6 +18,10 @@ const fetcher = (path, opts) => fetch(path, opts)
         if (resp.ok) {
             return resp;
         } else {
+            if (resp.status === 401) {
+                // Unauthorized, remove token
+                window.sessionStorage.getItem('token')
```
Comment (Dr-Horv) [inline src/client/backend.js]: How does getItem remove token? Is it a getter with side effects?

Comment (lindskogen) [inline src/client/backend.js]: No, just tired-stress-running-for-bus-brain at the wheel, good catch!

File: src/client/Admin.js (hunk 1 above)
Comment (lindskogen) [inline src/client/Admin.js]: My intention was for it to perform the request and verify that we're signed in.. It turned out to not really work..

**Pull request merged**